core-services/sanitize-prow-jobs/_config.yaml

```diff
@@ -3329,3 +3329,22 @@ groups:
     - openshift/pagerduty-operator/.*-presubmits\.yaml$
     - opendatahub-io/opendatahub-operator/.*-presubmits\.yaml$
     - cloud-bulldozer/plow/.*-presubmits\.yaml$
+  vsphere:
+    jobs:
+    - pull-ci-openshift-installer-fcos-e2e-vsphere-upi
+    - pull-ci-openshift-installer-master-e2e-vsphere-upi
+    - pull-ci-openshift-installer-release-4.5-e2e-vsphere-upi
+    - pull-ci-openshift-installer-release-4.6-e2e-vsphere-upi
+    - pull-ci-openshift-installer-release-4.7-e2e-vsphere-upi
+    - pull-ci-openshift-machine-config-operator-master-e2e-vsphere-upi
+    - pull-ci-openshift-machine-config-operator-release-4.5-e2e-vsphere-upi
+    - pull-ci-openshift-machine-config-operator-release-4.6-e2e-vsphere-upi
+    - pull-ci-openshift-machine-config-operator-release-4.7-e2e-vsphere-upi
+    - release-openshift-ocp-installer-e2e-vsphere-upi-4.4
```

Comment:
The release jobs i think still use templates ans this might have moved the release jobs to new VMC without updating the templates to use new credentials and configuration.